common/changes/@uifabric/styling/pickers-fixstackoverflow_2017-08-15-22-39.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@uifabric/styling",
+      "type": "none"
```

Comment:
minor suggestion, run `rush change -b upstream/master` to avoid adding the unnecessary change files.